app/models/user.rb

```diff
@@ -6,9 +6,7 @@ class User < ApplicationRecord
   validates :email, email: { strict_mode: true }, if: "uid.nil?", on: :create
 
   enum role: %w(customer venue_admin platform_admin)
-
-  after_create :send_welcome_email
```

Comment:
@ryanflach That would be my preference since having it in the model starts to violate SRP.

https://robots.thoughtbot.com/back-to-basics-solid#single-responsibility-principle